packages/generator/src/generator.js

```diff
@@ -349,7 +350,7 @@ export default class Generator {
         }
         // Add route to manifest (only if no error and redirect)
         if (this.manifest && (!res.error && !res.redirected)) {
-          this.manifest.routes.push(route)
+          this.manifest.routes.push(withoutTrailingSlash(route))
```

Comment:
I think we are missing `|| '/'` fallback here. It should be finally fixed back in ufo :D

Reply:
Added in 0.6.3